opennmt/utils/wer.py

```diff
@@ -0,0 +1,44 @@
+"""Hypotheses file scoring."""
+import numpy
+import pyter
+
+def wer(ref_path, hyp_path):
+  """ Compute Word Error Rate between two files """
+  ref_fp = open(ref_path)
+  hyp_fp = open(hyp_path)
```

Comment:
These file handles should be closed at the end of the function. To do it automatically, you could turn this into:

```python
with open(ref_path) as ref_fp, open(hyp_path) as hyp_fp:
  ...
```

This also applies for the other scoring functions.

Reply:
Good suggestion, fine by me